### DISCUSSION TITLE
Unit testing

### DISCUSSION THREAD

**Discussion by devsoftzz** [WebMetadataRetrieval/15-Web_Metadata_Retrieval#56] [General]
Django provides its own test module **django.test**
I found these videos for Unit testing of Django project
https://youtube.com/playlist?list=PLbpAWbHbi5rMF2j5n6imm0enrSD9eQUaM
https://youtu.be/1FqxfnlQPi8

**Comment by shivam1shroff** [comment]
I've also found another Module that helps us in Unit testing **unittest**
It is also an inbuilt module.
https://www.youtube.com/watch?v=6tNS--WetLI&t=1431s
